### DISCUSSION TITLE
Rename Column in Sequelize Entity Framework

### DISCUSSION THREAD

**Discussion by ibrar-uzair** [sequelize/sequelize#13814] [Help & Questions]
I just wanted to rename my Column in Entity but I couldn't find any way to solve this issue. Yeah, we can do {alter:true} but what it does is, it deletes the first column and then insert another column in the database. Even if we try {force:true}, it deletes the column data. Is there anyone who can help me with this issue? Yeah, we can solve this issue, when use Sequelize in other way like create migrations and seeders but can't do it here.

**Comment by fzn0x** [comment]
Sorry this is not entity framework community discussion, do you mean you need workaround from Sequelize? it's quite simple.

```js
// your migration-file.js
'use strict';

module.exports = {
  up: (queryInterface, Sequelize) => {
    return queryInterface.renameColumn('tableName', 'tour_title', 'tour_name');
  },

  down: (queryInterface, Sequelize) => {
    return queryInterface.renameColumn('tableName', 'tour_name', 'tour_title');
  }
};
```

**Reply by ibrar-uzair** [reply]
I want to rename the column without using migrations

**Reply by ephys** [reply]
@ibrar-uzair there isn't that many other ways. Your only options are:

- Using a migration.
- Manually running the `ALTER` query in your database.
- force-sync models (will result in data loss).
